src/libs/lv_conf.h

```diff
@@ -757,4 +757,4 @@ typedef void* lv_obj_user_data_t;
 
 /*--END OF LV_CONF_H--*/
 
-#endif /*LV_CONF_H*/
\ No newline at end of file
+#endif /*LV_CONF_H*/
```

Comment:
If you want to fully roll back the un-related LVGL changes, you also need to remove the newline at the end of the file. i.e. at the end of line 760, go to the end of the comment, and press delete so that line 761 is not present.